components/woo/order.php

```diff
@@ -1308,36 +1307,37 @@ public function get_recipient( $prefix = '' ) {
 		if ( '' === $recipient || 0 == count( $recipient ) ) {
 			$order = $this->get_wc_order();
 
-			$recipient_street_name = $order->shipping_address_1;
+			$recipient_street_name = method_exists( $order, 'get_shipping_address_1' ) ? $order->get_shipping_address_1() : $order->shipping_address_1;
```

Comment:
We merge this for now but in the long run we need a proper adapter.
Something that offers an interface we defined and internally maps to Woo2 or Woo3 so that we don't need to care anymore.

Reply:
Yeah I know. This can't be the best way to handle downward compatibility. 😉